timescaledb/quick-start/ruby.md

```diff
@@ -4,727 +4,708 @@ excerpt: Get started with TimescaleDB for a Ruby application
-database `tsdb` like so:
+1.  Update the contents of the `<XXXXXXXXXXXXXX>_add_timescale.rb` file with
+    these instructions to load the TimescaleDB extension to PostgreSQL:
 
-```ruby
-development:
-  <<: *default
-  database: tsdb
-```
+    ```ruby
+    class AddTimescale < ActiveRecord::Migration[7.0]
+      def change
+       create_table :page_loads, id: false do |t|
+          t.string :user_agent
 
-Repeat the step for the `test` and `production` sections further down this file.
+          t.timestamps
+       end
+      end
+     end
 
-Your final file should look like this (without all the automatically generated comments):
+     ```
 
-```yaml
-default: &default
-  adapter: postgresql
-  encoding: unicode
-  # For details on connection pooling, see Rails configuration guide
-  # http://guides.rubyonrails.org/configuring.html#database-pooling
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  host: [your hostname]
-  port: [your port]
-  username: [your username]
-  password: [your password]
+1.  Add the TimescaleDB extension to the PostgreSQL database:
 
-development:
-  <<: *default
-  database: tsdb
+    ```ruby
+       rails db:migrate
+    ```
 
-test:
-  <<: *default
-  database: tsdb
+1.  Connect to TimescalDB Cloud using Rails:
 
-production:
-  <<: *default
-  database: tsdb
-```
+    ```bash
+       echo "\dx" | rails dbconsole
+    ```
 
-#### Create the database
+    If the connection is successful, you are prompted for the password to
+    connect to TimescaleDB. Type the `<PASSWORD>` for `tsdbadmin` in the TimescaleDB Cloud.
 
-Now we can run the following `rake` command to create the database in TimescaleDB:
+    The result is similar to this:
+  
+    ```bash
+                                                           List of installed extensions
+        Name         | Version |   Schema   |                                      Description                                      
+        ---------------------+---------+------------+---------------------------------------------------------------------------------------
+        pg_stat_statements  | 1.10    | public     | track planning and execution statistics of all SQL statements executed
+        plpgsql             | 1.0     | pg_catalog | PL/pgSQL procedural language
+        timescaledb         | 2.9.3   | public     | Enables scalable inserts and complex queries for time-series data
+        timescaledb_toolkit | 1.13.1  | public     | Library of analytical hyperfunctions, time-series pipelining, and other SQL  utilities
+       (4 rows)
+    ```
 
-```bash
-rails db:create
-```
+      <highlight type="important">
+       To ensure that your tests run successfully, in the `config/environments/test.rb`
+       file, add `config.active_record.verify_foreign_keys_for_fixtures = false`.
+       Otherwise you get an error because TimescaleDB uses internal foreign keys.
+       </highlight>
```

Comment:
```suggestion
      <Highlight type="important">
       To ensure that your tests run successfully, in the `config/environments/test.rb`
       file, add `config.active_record.verify_foreign_keys_for_fixtures = false`.
       Otherwise you get an error because TimescaleDB uses internal foreign keys.
       </Highlight>
```